transports/transport-rest/transport-rest-client/src/main/java/org/apache/servicecomb/transport/rest/client/http/DefaultHttpClientFilter.java

```diff
@@ -79,13 +82,15 @@ protected Object extractResult(Invocation invocation, HttpServletResponseEx resp
               responseEx.getStatus(),
               responseEx.getStatusType().getReasonPhrase(),
               responseEx.getHeader(HttpHeaders.CONTENT_TYPE));
+      LOGGER.error(msg);
       return ExceptionFactory.createConsumerException(new InvocationException(responseEx.getStatus(), responseEx.getStatusType().getReasonPhrase(), msg));
     }
 
     try {
       return produceProcessor.decodeResponse(responseEx.getBodyBuffer(), responseMeta.getJavaType());
     } catch (Exception e) {
-      return ExceptionFactory.createConsumerException(e);
+      LOGGER.error("failed to decode response body", e);
+      throw ExceptionFactory.createConsumerException(e);
```

Comment:
change "extractResult" to "extractResponse"?

Reply:
OK, please review again.

Reply:
oh, i means result of the method change form "result" to "Response"
otherwise line 86 still have problem.

Reply:
Get it...... Maybe we can  throw the `InvocationException` directly? After all, a none 2xx response will be transferred into an `Exception` in `Invoker.syncInvoke()` and `CseClientHttpRequest.invoke()`.